content/infra_language/secrets.md

```diff
@@ -11,19 +11,13 @@ gh_repo = "chef-web-docs"
     parent = "chef_infra/infra_language"
 +++
 
-Chef Infra Client 17.5 and later includes a beta Secrets Management Integration helper. This helper allows accessing secrets from the following secrets management systems within your Infra recipes or resources:
+Chef Infra Client 17.5 and later includes a Secrets Management Integration helper. This helper allows accessing secrets from the following secrets management systems within your Infra recipes or resources:
```

Comment:
We might keep a note saying that it's beta for < 18 since we'll have customers using < 18 for a few more years.

```suggestion
The Secrets Management Integration helper is a beta feature starting in Chef Infra Client 17.5 and became a fully supported feature in Chef Infra Client 18. This helper allows accessing secrets from the following secrets management systems within your Infra recipes or resources:
```

Reply:
Thanks for the feedback @IanMadd 
In case Secret Manager, since there hasn't been any new changes added and the feature is thoroughly tested, we intend to inform people it is safe to use it, even in 17, without worrying about breaking changes getting introduced.

Reply:
Hi @IanMadd, turns out we will keep the suggestion as you had mentioned, saying it being beta for chef 17